include/deal.II/base/vector_slice.h

```diff
@@ -156,122 +159,106 @@ make_slice (VectorType &v)
  * @author Guido Kanschat, 2004
  */
 template <typename VectorType>
-inline
-const VectorSlice<const VectorType>
-make_slice (VectorType         &v,
-            const unsigned int start,
-            const unsigned int length)
+inline const VectorSlice<const VectorType>
+make_slice(VectorType &v, const unsigned int start, const unsigned int length)
 {
   const VectorSlice<const VectorType> r(v, start, length);
   return r;
 }
 
 
 
-
 //---------------------------------------------------------------------------
 
 template <typename VectorType>
-inline
-VectorSlice<VectorType>::VectorSlice(VectorType &v)
-  :
-  v(v), start(0), length(v.size())
+inline VectorSlice<VectorType>::VectorSlice(VectorType &v)
+  : v(v), start(0), length(v.size())
```

Comment:
we've usually detached the colon from the member initialization

Reply:
So time for a new option to vote on.

Reply:
Yes :-)

Reply:
See https://github.com/masterleinad/dealii/pull/24 for the other option.